<?xml version="1.0"?>
<launch>
  <param name="pub_map_odom_transform" value="true"/> 
  <param name="map_frame" value="map"/> 
  <param name="base_frame" value="base_frame"/> 
  <param name="odom_frame" value="odom"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_0" args="-0.50 0.50 0 0 0 0 /map robot_0/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_1" args="-1.00 0.50 0 0 0 0 /map robot_1/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_2" args="-1.50 0.50 0 0 0 0 /map robot_2/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_3" args="-2.00 0.50 0 0 0 0 /map robot_3/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_4" args="-2.50 0.50 0 0 0 0 /map robot_4/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_5" args="-3.00 0.50 0 0 0 0 /map robot_5/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_6" args="-3.00 1.00 0 0 0 0 /map robot_6/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_7" args="-3.00 1.50 0 0 0 0 /map robot_7/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_8" args="-3.00 2.00 0 0 0 0 /map robot_8/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_9" args="-3.00 2.50 0 0 0 0 /map robot_9/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_10" args="-3.00 3.00 0 0 0 0 /map robot_10/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_11" args="-2.50 3.00 0 0 0 0 /map robot_11/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_12" args="-2.00 3.00 0 0 0 0 /map robot_12/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_13" args="-1.50 3.00 0 0 0 0 /map robot_13/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_14" args="-1.00 3.00 0 0 0 0 /map robot_14/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_15" args="-0.50 3.00 0 0 0 0 /map robot_15/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_16" args="-0.50 2.50 0 0 0 0 /map robot_16/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_17" args="-0.50 2.00 0 0 0 0 /map robot_17/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_18" args="-0.50 1.50 0 0 0 0 /map robot_18/odom 100"/>
  <node pkg="tf" type="static_transform_publisher" name="robot_19" args="-0.50 1.00 0 0 0 0 /map robot_19/odom 100"/>
</launch>